optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/roster/RosterState.java

```diff
@@ -103,21 +103,20 @@ public boolean isHistoric(Shift shift) {
         return shift.getEndDateTime().isBefore(OffsetDateTime.of(getFirstPublishedDate().atTime(LocalTime.MIDNIGHT), shift.getEndDateTime().getOffset()));
```

Comment:
Should a shift that is in progress be historic?

Reply:
Good question. For now, let's always use the startDateTime to decide if a shift is historic/published/draft. But once I see some end-users of this, I'll ask whether or not that makes sense that shifts that are in progress are considered historic in visualization.
I'd argue: yes, because for non-disruptive replanning of published shifts (!= our current "solve" button) shifts that have already started can't change, so those are historic.